### PR TITLE
11.0 mig website sale hide price

### PR DESCRIPTION
**Pull request by AbrahamTrey**
[MIG] website_sale_hide_price: Migration to 11.0

### CODE REVIEW

File: website_sale_hide_price/models/__init__.py [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
```
Comment (chienandalu) [inline website_sale_hide_price/models/__init__.py]: This is not needed in python 3

File: website_sale_hide_price/models/__init__.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
...
-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
```
Comment (chienandalu) [inline website_sale_hide_price/models/__init__.py]: Why adding this? Delete it, please

File: website_sale_hide_price/models/res_partner.py [hunk 3]
```diff
@@ -10,5 +10,4 @@ class ResPartner(models.Model):
 
     website_show_price = fields.Boolean(
         string='Show prices on website',
-        default=True,
-    )
+        default=True)
```
Comment (chienandalu) [inline website_sale_hide_price/models/res_partner.py]: Don't change coding style unnecesarily

File: website_sale_hide_price/models/website.py [hunk 4]
```diff
@@ -16,3 +17,5 @@ def _compute_website_show_price(self):
         for rec in self:
             rec.website_show_price = (
                 request.env.user.partner_id.website_show_price)
+
```
Comment (chienandalu) [inline website_sale_hide_price/models/website.py]: :fire:

File: website_sale_hide_price/models/website.py [hunk 5]
```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 # Copyright 2017 Tecnativa - David Vidal
+# Copyright 2018 Trey, kilobytes de soluciones - Abraham Gonzalez
```
Comment (chienandalu) [inline website_sale_hide_price/models/website.py]: Credit yourself just if you make remarkable changes to the file.

File: website_sale_hide_price/__init__.py [hunk 6]
```diff
@@ -1,3 +1,2 @@
 # -*- coding: utf-8 -*-
```
Comment (chienandalu) [inline website_sale_hide_price/__init__.py]: :fire:

File: website_sale_hide_price/__manifest__.py [hunk 7]
```diff
@@ -1,11 +1,13 @@
 # -*- coding: utf-8 -*-
 # Copyright 2017 Tecnativa - David Vidal
+# Copyright 2018 Trey, kilobytes de soluciones - Abraham Gonzalez
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     'name': 'Website Sale Hide Price',
     'version': '10.0.1.0.0',
```
Comment (chienandalu) [inline website_sale_hide_price/__manifest__.py]: 11.0.1.0.0

File: website_sale_hide_price/__manifest__.py [hunk 8]
```diff
@@ -1,11 +1,13 @@
...
     'category': 'Website',
-    'author': 'Tecnativa, '
+    'author': 'Trey, Kilobytes de soluciones, '
+              'Tecnativa, '
               'Odoo Community Association (OCA)',
     'website': 'https://www.tecnativa.com',
```
Comment (chienandalu) [inline website_sale_hide_price/__manifest__.py]: https://github.com/OCA/e-commerce

File: website_sale_hide_price/__manifest__.py [hunk 9]
```diff
@@ -1,11 +1,13 @@
 # -*- coding: utf-8 -*-
 # Copyright 2017 Tecnativa - David Vidal
+# Copyright 2018 Trey, kilobytes de soluciones - Abraham Gonzalez
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     'name': 'Website Sale Hide Price',
     'version': '10.0.1.0.0',
     'category': 'Website',
-    'author': 'Tecnativa, '
+    'author': 'Trey, Kilobytes de soluciones, '
```
Comment (chienandalu) [inline website_sale_hide_price/__manifest__.py]: As no remarkable changes have been done in the migration, your credits should go to the readme

**Review by chienandalu — CHANGES_REQUESTED** [review]
Thanks for the work! 

Please use the new OCA readme system: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: website_sale_hide_price/__manifest__.py [hunk 10]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - David Vidal
+# Copyright 2018 Trey, kilobytes de soluciones - Abraham Gonzalez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Website Sale Hide Price',
+    'version': '10.0.1.0.0',
+    'category': 'Website',
+    'author': 'Trey, Kilobytes de soluciones, '
```
Comment (pedrobaeza) [inline website_sale_hide_price/__manifest__.py]: You shouldn't add your company here unless you make significant changes on the module, which is not the case.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@chienandalu please finish your review

**Review by wtaferner — APPROVED** [review]

**Pull request merged**